src/pages/Detail/Aside/CheckBox.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CheckBox = ({ toggleCheckbox }) => {
+  return (
+    <CheckBoxSection>
+      <div>
+        <input type="checkbox" onChange={toggleCheckbox} />
+        <label>단독홀</label>
+      </div>
+      <Price fontSize="17px">
+        ₩6000
+        <Text>/ 시간</Text>
+      </Price>
+    </CheckBoxSection>
+  );
+};
+
+export default CheckBox;
+
+const CheckBoxSection = styled.div`
+  ${props => props.theme.flexMixIn('space-between', 'center')}
+  padding: 10px 5px;
+  font-size: 18px;
+
+  label {
+    margin-left: 10px;
+  }
+`;
+
+const Price = styled.h3`
+  color: ${props => props.theme.purple};
+  font-size: ${props => props.fontSize};
```

Comment:
구조 분해 할당을 해줄 수 있습니다!
```suggestion
  font-size: ${({fontSize}) => fontSize};
```